rand_distr/src/normal_inverse_gaussian.rs

```diff
@@ -0,0 +1,107 @@
+use crate::{Distribution, Float, InverseGaussian, Standard, StandardNormal};
+use rand::Rng;
+
+/// Error type returned from `NormalInverseGaussian::new`
+#[derive(Debug, PartialEq)]
+pub enum Error {
+    /// `alpha <= 0` or `nan`.
+    AlphaNegativeOrNull,
+    /// `|beta| >= alpha` or `nan`.
+    AbsoluteBetaLessThanAlpha,
```

Comment:
Should be `AbsoluteBetaNotLessThanAlpha` ?

Reply:
Yes good catch !